### ISSUE TITLE
Null pointer exception when trying to open Theme search fragment

### ISSUE THREAD

**Issue by kwonye** [wordpress-mobile/WordPress-Android#3983]
No known repro steps

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.themes.ThemeBrowserActivity}: java.lang.NullPointerException
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2067)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2092)
       at android.app.ActivityThread.access$600(ActivityThread.java:133)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1203)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4812)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:792)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:559)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.NullPointerException
       at android.support.v4.view.MenuItemCompatHoneycomb.getActionView(MenuItemCompatHoneycomb.java:39)
       at android.support.v4.view.MenuItemCompat$HoneycombMenuVersionImpl.getActionView(MenuItemCompat.java:176)
       at android.support.v4.view.MenuItemCompat.getActionView(MenuItemCompat.java:325)
       at org.wordpress.android.ui.themes.ThemeSearchFragment.configureSearchView(ThemeSearchFragment.java:87)
       at org.wordpress.android.ui.themes.ThemeSearchFragment.restoreState(ThemeSearchFragment.java:59)
       at org.wordpress.android.ui.themes.ThemeSearchFragment.onCreateView(ThemeSearchFragment.java:45)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:829)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1035)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1017)
       at android.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:1802)
       at android.app.Activity.performCreate(Activity.java:5205)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1094)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2031)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2092)
       at android.app.ActivityThread.access$600(ActivityThread.java:133)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1203)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4812)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:792)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:559)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by tonyr59h** [comment]
Addressed in #4108, closing.

**Comment by kwonye** [comment]
Thanks @maxme, I'll look into another fix.
